notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>Also, often you may see <code>16/16</code> version being faster than <code>16/8</code>. </blockquote><p>For user readability, can we please define what these ratios are depicting? You've got it in the code above but as a first-time reader of the notebook, it wasn't immediately obviously to me and it's simple enough to say something in the text like "You'll notice the datapoints are labeled as a/b, which corresponds to the number of bits from<code>internal_distance_dtype/lut_dtype</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I still don't see these defined anywhere. Can you expend a few words to explain what is the numerator and what is the denominator please?</p>

Reply:
<p>I've added it directly to the plot, is that fine?</p>